src/main/java/hudson/plugins/testlink/AbstractTestLinkBuilder.java

```diff
@@ -182,13 +188,13 @@ public AbstractTestLinkBuilder(String testLinkName, String testProjectName, Stri
      * @deprecated
      */
     public AbstractTestLinkBuilder(String testLinkName, String testProjectName, String testPlanName,
-            String platformName, String buildName, String customFields, Boolean executionStatusNotRun,
+            String platformName, String buildName, String customFields, String testPlanCustomFields, Boolean executionStatusNotRun,
```

Comment:
It is not specifically for Java. But XStream, the library used to serialise XML in Jenkins, may try to load an XML or call the old constructor.

So we keep it for a while, just until users update their jobs.

A major release like 4.0 is a good opportunity to remove this deprecated constructor, giving notice with enough time to users too.